### PR TITLE
Added Cloud Convert

### PR DESCRIPTION
**Pull request by hebinzin**

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -1421,7 +1421,18 @@
             "cl.ly"
         ]
     },
-
+
```
Comment (tupaschoal) [inline _data/sites.json:1424]:
```suggestion

```

File: _data/sites.json [hunk 2]
```diff
@@ -1421,7 +1421,18 @@
             "cl.ly"
         ]
     },
-
+    
+    {
+        "name": "Cloud Convert",
```
Comment (tupaschoal) [inline _data/sites.json:1426]: Because of the whitespace, Cloud Convert needs to come before Cloud App

File: _data/sites.json [hunk 3]
```diff
@@ -1421,7 +1421,18 @@
...
+        "url": "https://cloudconvert.com/delete#",
+        "difficulty": "easy",
+        "notes": "Go to the dashboard > account > details, and fill your username and password. The acount is deleted after 72 hours.",
+        "notes_pt_br": "Vá em \"dashboard\" > \"account\" > \"details\", e preencha seu nome de usuário e senha. A conta é removida após 72 horas.",
+        "domains": [
+            "cloudconvert.com"
+        ]
+    },
+
```
Comment (tupaschoal) [inline _data/sites.json:1435]:
```suggestion

```

**Comment by hebinzin** [comment]
Please, some noob help here, I'm not sure what's the best way to apply suggestions. Should I make the changes in my repo and make another pull request?

**Comment by tupaschoal** [comment]
I see you pushed to your own repo, on the master branch (as noted above with `hebinzin:master`. If you do another commit there applying those suggestions, they will be automatically pulled into this pull request, because a pull request means "apply all changes from this branch onto your branch", in your case "apply all changes from my master into `jdm-contrib:master`"

So short answer: just fix then in your repo and it'll be fine.

**Comment by hebinzin** [comment]
Nice, thank you! I think they're all fixed now.

**Comment by tupaschoal** [comment]
Looks good, thanks for your contribution.

Just a tip for the future: when you work with git, always make sure you make your changes on a new branch, not on `master` or `main`, it might prevent some headaches

**Comment by hebinzin** [comment]
Thanks for the advice!

**Pull request merged**